tests/testthat/test_cage.R

```diff
@@ -22,7 +22,7 @@ start(cageEqualStart)[3] <- start(cageEqualStart)[3] - 1
 
 test_that("reassignTSSbyCage picks best one max peak of several", {
   # second granges have higher score, and both are within frame, then max one should be picked
-  test_result <- reassignTSSbyCage(fiveUTRs[1], cage = cageEqualStart, cds = cds)
+  test_result <- suppressWarnings(reassignTSSbyCage(fiveUTRs[1], cage = cageEqualStart, cds = cds))
```

Comment:
no, this is not the correct fix for min or max

Reply:
This is not for max min, this a bug in GenomicFeatures which will be fixed in bioC 3.7. We can remove it then.

Reply:
I thought you are working with the devel version of bioC 3.7 as you should if you want to push it to this bioc version! if this is true we dont need suppressWarnings at all and it all should be working just fine

Reply:
I'm using 3.6, should I change now ?